<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>292</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>shutdown</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="Log_Text">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>521</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="autocon">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>95</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>自动连接</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connect2server">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>92</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>10</y>
      <width>229</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>设备ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>主题</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="clientID"/>
       </item>
       <item>
        <widget class="QLineEdit" name="topic"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>584</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="version">
    <property name="title">
     <string>版本</string>
    </property>
    <addaction name="timeshow"/>
   </widget>
   <addaction name="version"/>
  </widget>
  <action name="actiontime">
   <property name="text">
    <string>time</string>
   </property>
  </action>
  <action name="timeshow">
   <property name="text">
    <string>time</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
